addon/mixins/page-speed-mixin.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+const { newrelic } = window;
+
+export default Ember.Mixin.create({
+  _startTiming: Ember.on('didTransition', function() {
+    Ember.run.scheduleOnce('afterRender', this, '_finishTiming');
+  }),
+
+  _finishTiming() {
+    if (newrelic && performance) {
+      let seconds = (Date.now() - performance.timing.domComplete) / 1000;
+      newrelic.addPageAction(this.get('url'), seconds);
```

Comment:
would you want to send the time between the last transition finish and this one?

for the first transition, you would want to measure relative to domComplete.

but for subsequent times, you should save the value of `Date.now()` and using that for measuring the next transition.

otherwise, as you navigate around the app, the `seconds` will get larger and larger even though transition was fast.